### DISCUSSION TITLE
How to specify a specific domain to use dnscrypt-proxy, but others use the default dns server

### DISCUSSION THREAD

**Discussion by newmacwkl** [DNSCrypt/dnscrypt-proxy#1957] [Q&A]
How to specify a specific domain to use dnscrypt-proxy, but others use the default dns server

**Comment by BrianG61UK** [comment]
That's something to configure in your existing software, not in dnscrypt-proxy.
For instance, you can configure dnsmasq to forward certain queries to different servers, one of which could be dnscrypt-proxy.

**Comment by vasundhar** [comment]
If you use dnsmasq, it can be achieved https://serverfault.com/a/391926/82767
server=/specificdomain/127.0.0.1:'Your DNSCrypt Port'

**Comment by bcookatpcsd** [comment]
dnscrypt-proxy has example-forwarding-rules.txt.. 

that might be what your looking for without other software..
